src/main/java/com/revature/services/JMeterService.java

```diff
@@ -35,10 +37,13 @@
 
     // replace user with username later
     public static final String BASE_FILE_PATH = "./datafiles/user_";
+    
+    public static final String PROPERTIES_PATH = "src/test/resources/test.properties";
```

Comment:
If we have this as a final String, should probably make add a proper jmeter.properties to src/main/resources